src/components/separators/_horizontal-separator.scss

```diff
@@ -20,6 +20,10 @@ $includeHtml: false !default;
         margin: (rhythm(1) - $horizontalSeparatorHeight) 0 rhythm(1);
       }
     }
+
+    &--small-spaced {
```

Comment:
does it match our naming convention? BTW where the hell is our naming convention? I thought it's in wiki, but no.